#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_simplify_ui.patch by  <stgraber@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove the tryicon from ICA and hide info-bubble by default

@DPATCH@
diff -Nrup ../../italc-1.0.9-rc1.orig/ima/src/tool_button.cpp ../ima/src/tool_button.cpp
--- ../../italc-1.0.9-rc1.orig/ima/src/tool_button.cpp	2008-04-30 16:43:15.000000000 +0200
+++ ../ima/src/tool_button.cpp	2008-05-09 20:07:41.000000000 +0200
@@ -42,7 +42,7 @@
 const int MARGIN = 10;
 const int ROUNDED = 2000;
 
-bool toolButton::s_toolTipsDisabled = FALSE;
+bool toolButton::s_toolTipsDisabled = TRUE;
 bool toolButton::s_iconOnlyMode = FALSE;
 
 
